packages/ui-components/src/components/Map/Legend/Legend.js

```diff
@@ -21,7 +21,7 @@ const LegendFrame = styled.div`
   display: flex;
   width: fit-content;
   padding: 0.6rem;
-  margin: ${p => p.$spaceBetween || '0.6rem'} auto;
+  margin: 0.6rem auto;
```

Comment:
Minor: is it better to specify options on presentation components e.g. layout: 'vertical' | 'horizontal' or to allow custom components to be passed in? Happy with either way, just curious. Either way this seems better than spaceBetween

Reply:
Honestly I'm not sure, I think it's just a flavour that different devs have preferences for. I think the ideal is that you compose it together at the top level, the way that `mui` is set up, but that ends up requiring a lot more work and larger JSX at that top level than just providing "slots" that you can pass custom components through.